### PR TITLE
Update influxdb 1.0 to 1.0.0-beta2

### PR DESCRIPTION
**Pull request by jsternberg**
Update influxdata libraries to the new bashbrew library format

### CODE REVIEW

File: library/influxdb [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+GitRepo: git://github.com/influxdata/influxdb-docker
+GitCommit: 6d869aa598baf9d23019682ecff42d022a00ce17
+Directory: 0.12
 
-0.13: git://github.com/influxdata/influxdata-docker@215cf009c143dc739b5b10084ae330ca7f3665d6 influxdb/0.13
-0.13.0: git://github.com/influxdata/influxdata-docker@215cf009c143dc739b5b10084ae330ca7f3665d6 influxdb/0.13
-latest: git://github.com/influxdata/influxdata-docker@215cf009c143dc739b5b10084ae330ca7f3665d6 influxdb/0.13
+Tags: 0.13, 0.13.0, latest
+GitRepo: git://github.com/influxdata/influxdata-docker
+GitCommit: 5b975e66d1c36d87396256947db0c745b43844a1
```
Comment (tianon) [inline library/influxdb]: Is this the correct commit?  https://travis-ci.org/docker-library/official-images/builds/138948965#L446-L456

Comment (jsternberg) [inline library/influxdb]: No, I missed replacing that one. Nice catch.

**Comment by tianon** [comment]
```diff
warning: insecure protocol git:// detected: git://github.com/influxdata/influxdata-docker
diff --git a/influxdb_1.0.0-beta1/alpine/Dockerfile b/influxdb_1.0.0-beta2-alpine/Dockerfile
similarity index 97%
rename from influxdb_1.0.0-beta1/alpine/Dockerfile
rename to influxdb_1.0.0-beta2-alpine/Dockerfile
index bb96ec8..567e177 100644
--- a/influxdb_1.0.0-beta1/alpine/Dockerfile
+++ b/influxdb_1.0.0-beta2-alpine/Dockerfile
@@ -1,6 +1,6 @@
 FROM alpine:3.4
 
-ENV INFLUXDB_VERSION 1.0.0-beta1
+ENV INFLUXDB_VERSION 1.0.0-beta2
 RUN apk add --no-cache --virtual .build-deps wget gnupg tar ca-certificates && \
     update-ca-certificates && \
     gpg --keyserver hkp://ha.pool.sks-keyservers.net \
diff --git a/influxdb_1.0.0-beta1/alpine/entrypoint.sh b/influxdb_1.0.0-beta2-alpine/entrypoint.sh
similarity index 100%
rename from influxdb_1.0.0-beta1/alpine/entrypoint.sh
rename to influxdb_1.0.0-beta2-alpine/entrypoint.sh
diff --git a/influxdb_1.0.0-beta1/influxdb.conf b/influxdb_1.0.0-beta2-alpine/influxdb.conf
similarity index 100%
rename from influxdb_1.0.0-beta1/influxdb.conf
rename to influxdb_1.0.0-beta2-alpine/influxdb.conf
diff --git a/influxdb_1.0.0-beta1/Dockerfile b/influxdb_1.0.0-beta2/Dockerfile
similarity index 93%
rename from influxdb_1.0.0-beta1/Dockerfile
rename to influxdb_1.0.0-beta2/Dockerfile
index f2fa0e9..0bae0c1 100644
--- a/influxdb_1.0.0-beta1/Dockerfile
+++ b/influxdb_1.0.0-beta2/Dockerfile
@@ -4,7 +4,7 @@ RUN gpg \
     --keyserver hkp://ha.pool.sks-keyservers.net \
     --recv-keys 05CE15085FC09D18E99EFB22684A14CF2582E0C5
 
-ENV INFLUXDB_VERSION 1.0.0-beta1
+ENV INFLUXDB_VERSION 1.0.0-beta2
 RUN wget -q https://dl.influxdata.com/influxdb/releases/influxdb_${INFLUXDB_VERSION}_amd64.deb.asc && \
     wget -q https://dl.influxdata.com/influxdb/releases/influxdb_${INFLUXDB_VERSION}_amd64.deb && \
     gpg --batch --verify influxdb_${INFLUXDB_VERSION}_amd64.deb.asc influxdb_${INFLUXDB_VERSION}_amd64.deb && \
@@ -12,7 +12,7 @@ RUN wget -q https://dl.influxdata.com/influxdb/releases/influxdb_${INFLUXDB_VERS
     rm -f influxdb_${INFLUXDB_VERSION}_amd64.deb*
 COPY influxdb.conf /etc/influxdb/influxdb.conf
 
-EXPOSE 8083 8086
+EXPOSE 8086
 
 VOLUME /var/lib/influxdb
 
diff --git a/influxdb_1.0.0-beta1-alpine/Dockerfile b/influxdb_1.0.0-beta2/alpine/Dockerfile
similarity index 97%
rename from influxdb_1.0.0-beta1-alpine/Dockerfile
rename to influxdb_1.0.0-beta2/alpine/Dockerfile
index bb96ec8..567e177 100644
--- a/influxdb_1.0.0-beta1-alpine/Dockerfile
+++ b/influxdb_1.0.0-beta2/alpine/Dockerfile
@@ -1,6 +1,6 @@
 FROM alpine:3.4
 
-ENV INFLUXDB_VERSION 1.0.0-beta1
+ENV INFLUXDB_VERSION 1.0.0-beta2
 RUN apk add --no-cache --virtual .build-deps wget gnupg tar ca-certificates && \
     update-ca-certificates && \
     gpg --keyserver hkp://ha.pool.sks-keyservers.net \
diff --git a/influxdb_1.0.0-beta1-alpine/entrypoint.sh b/influxdb_1.0.0-beta2/alpine/entrypoint.sh
similarity index 100%
rename from influxdb_1.0.0-beta1-alpine/entrypoint.sh
rename to influxdb_1.0.0-beta2/alpine/entrypoint.sh
diff --git a/influxdb_1.0.0-beta1/alpine/influxdb.conf b/influxdb_1.0.0-beta2/alpine/influxdb.conf
similarity index 100%
rename from influxdb_1.0.0-beta1/alpine/influxdb.conf
rename to influxdb_1.0.0-beta2/alpine/influxdb.conf
diff --git a/influxdb_1.0.0-beta1/entrypoint.sh b/influxdb_1.0.0-beta2/entrypoint.sh
similarity index 100%
rename from influxdb_1.0.0-beta1/entrypoint.sh
rename to influxdb_1.0.0-beta2/entrypoint.sh
diff --git a/influxdb_1.0.0-beta1-alpine/influxdb.conf b/influxdb_1.0.0-beta2/influxdb.conf
similarity index 100%
rename from influxdb_1.0.0-beta1-alpine/influxdb.conf
rename to influxdb_1.0.0-beta2/influxdb.conf
```

**Comment by tianon** [comment]
LGTM

Build test of #1860; efb61fa (`chronograf`, `influxdb`, `kapacitor`, `telegraf`):

```console

$ bashbrew build chronograf:0.12
Using bashbrew/cache:923fb35af9147730bbf639beb7f77bb62034ece923f9464129bdc88126804495 (chronograf:0.12)
Tagging chronograf:0.12
Tagging chronograf:0.12.0

$ test/run.sh chronograf:0.12
testing chronograf:0.12
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build chronograf:0.13
Using bashbrew/cache:d8e4a1df9f32de9b28b95ca839f1a88cbc4a8a18ef83d6a2519ce2c07ee21d80 (chronograf:0.13)
Tagging chronograf:0.13
Tagging chronograf:0.13.0
Tagging chronograf:latest

$ test/run.sh chronograf:0.13
testing chronograf:0.13
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

```console

$ bashbrew build influxdb:0.12
Using bashbrew/cache:7f83a08a6ab1582f766aa68b3b93caa39fceb9a331269a94fc9153b4b8c2f277 (influxdb:0.12)
Tagging influxdb:0.12
Tagging influxdb:0.12.2

$ test/run.sh influxdb:0.12
testing influxdb:0.12
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build influxdb:0.13
Using bashbrew/cache:f6adf14c6f7491e2a425de6a1da8acb28199b6d1ec218f837d9adc411ee48a1f (influxdb:0.13)
Tagging influxdb:0.13
Tagging influxdb:0.13.0
Tagging influxdb:latest

$ test/run.sh influxdb:0.13
testing influxdb:0.13
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build influxdb:0.13-alpine
Using bashbrew/cache:8bed41e0149847bd21c1054c35e7585edcb424c568e5998b9c2ce8acfa51e97b (influxdb:0.13-alpine)
Tagging influxdb:0.13-alpine
Tagging influxdb:0.13.0-alpine
Tagging influxdb:alpine

$ test/run.sh influxdb:0.13-alpine
testing influxdb:0.13-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build influxdb:1.0.0-beta2
Using bashbrew/cache:b38ff89a28a554cc0f0d51f90e9031dc80d343cc683847028f4f94085069cf24 (influxdb:1.0.0-beta2)
Tagging influxdb:1.0.0-beta2

$ test/run.sh influxdb:1.0.0-beta2
testing influxdb:1.0.0-beta2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build influxdb:1.0.0-beta2-alpine
Using bashbrew/cache:5da46937034713e67c7b7608fe774dcbc97e6ac306bef8581e7ac691f56fb85f (influxdb:1.0.0-beta2-alpine)
Tagging influxdb:1.0.0-beta2-alpine

$ test/run.sh influxdb:1.0.0-beta2-alpine
testing influxdb:1.0.0-beta2-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

```console

$ bashbrew build kapacitor:0.12
Using bashbrew/cache:35718eb7ffbf46daefb5ab45a30f8560fe683aa7dfb632fc359223ec3edace0b (kapacitor:0.12)
Tagging kapacitor:0.12
Tagging kapacitor:0.12.0

$ test/run.sh kapacitor:0.12
testing kapacitor:0.12
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kapacitor:0.13
Using bashbrew/cache:099a8b8e8234d7db9d553cd778796f9bcc15513f1a901152dbd15b225db81504 (kapacitor:0.13)
Tagging kapacitor:0.13
Tagging kapacitor:0.13.1
Tagging kapacitor:latest

$ test/run.sh kapacitor:0.13
testing kapacitor:0.13
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kapacitor:0.13-alpine
Using bashbrew/cache:31f7da4f925a7c282351cfdf1665db560dd790cf20f637d5ad47fb0d577f2a62 (kapacitor:0.13-alpine)
Tagging kapacitor:0.13-alpine
Tagging kapacitor:0.13.1-alpine
Tagging kapacitor:alpine

$ test/run.sh kapacitor:0.13-alpine
testing kapacitor:0.13-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kapacitor:1.0.0-beta1
Using bashbrew/cache:0e09618d7875e23f914cbad690d111af632e0578df9cf758a814c02b07c4038b (kapacitor:1.0.0-beta1)
Tagging kapacitor:1.0.0-beta1

$ test/run.sh kapacitor:1.0.0-beta1
testing kapacitor:1.0.0-beta1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kapacitor:1.0.0-beta1-alpine
Using bashbrew/cache:0c08dbb36bfb6fe35f3e520750174cd4f913cfc329b467156dc04b9c90352e94 (kapacitor:1.0.0-beta1-alpine)
Tagging kapacitor:1.0.0-beta1-alpine

$ test/run.sh kapacitor:1.0.0-beta1-alpine
testing kapacitor:1.0.0-beta1-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

```console

$ bashbrew build telegraf:0.12
Using bashbrew/cache:d0e7ce4a5d7495bd8d2769f0edb1d7ce4cf790c92bdddc36e7f288da9bdf51e5 (telegraf:0.12)
Tagging telegraf:0.12
Tagging telegraf:0.12.0

$ test/run.sh telegraf:0.12
testing telegraf:0.12
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build telegraf:0.13
Using bashbrew/cache:ac947dd839d695ab9f975fd2c119613eca91492110451b54d36b796c5e660822 (telegraf:0.13)
Tagging telegraf:0.13
Tagging telegraf:0.13.1
Tagging telegraf:latest

$ test/run.sh telegraf:0.13
testing telegraf:0.13
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...	passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build telegraf:0.13-alpine
Using bashbrew/cache:8871799db7cbe613c2445f5a3f6be94b43bfa607cac37dabdfb5fa81fcf6d32f (telegraf:0.13-alpine)
Tagging telegraf:0.13-alpine
Tagging telegraf:0.13.1-alpine
Tagging telegraf:alpine

$ test/run.sh telegraf:0.13-alpine
testing telegraf:0.13-alpine
	'utc' [1/4]...]passed
	'cve-2014--shellshock'passed.. 
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build telegraf:1.0.0-beta1
Using bashbrew/cache:29ef08e6192ef5082faf096b90911d1e18d8c3e2a6885e725eebef2479ae9c8c (telegraf:1.0.0-beta1)
Tagging telegraf:1.0.0-beta1

$ test/run.sh telegraf:1.0.0-beta1
testing telegraf:1.0.0-beta1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build telegraf:1.0.0-beta1-alpine
Using bashbrew/cache:797cc318f0e6ffe16d140830567779276292f75497cef4b4c8aac16e805d5433 (telegraf:1.0.0-beta1-alpine)
Tagging telegraf:1.0.0-beta1-alpine

$ test/run.sh telegraf:1.0.0-beta1-alpine
testing telegraf:1.0.0-beta1-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**